content/VR/use-teleop/start/index.md

```diff
@@ -21,29 +21,26 @@ toc: true
 
 1. Make sure the robot is turned on, connected to the network and that all the robot's services are running before launching the teleoperation application.
```

Comment:
Maybe we can add a ref to the section on how to get a service status (/advanced/services/manage-services/#get-a-service-status)?